pages/cosmwasm/execute-update-price-feeds.mdx

```diff
@@ -0,0 +1,117 @@
+import Input from "../../components/Input";
+import Arg from "../../components/Arg";
+import Example from "../../components/Example";
+import Examples from "../../components/Examples";
+import DynamicCode from "../../components/DynamicCode";
+import { InputFormats } from "../../utils/InputFormat";
+import CosmWasmExecute from "../../components/CosmwasmExecute";
+import { Tab, Tabs } from "nextra-theme-docs";
+
+# Update Price Feeds
+
+Update the on-chain price feeds using the provided `data`, which contains serialized and signed price update data from Pyth Network.
+You can retrieve the latest price `data` for a given set of price feeds from the price service API.
+Executing this message updates the on-chain price if the provided update is more recent than the current on-chain price.
+Otherwise, the provided update will be ignored.
+Execution will succeed even if the update is ignored.
+
+This operation requires the caller to pay a fee to perform the update.
+The required fee for a given set of updates can be computed using the [GetUpdateFee](query-get-update-fee) query.
+
+Reverts if the required fee is not paid, or the `data` is incorrectly signed or formatted.
+
+<div className="mt-6 overflow-x-auto">
+
+| Argument                                           | Input                                           | Description                                                |
+| -------------------------------------------------- | ----------------------------------------------- | ---------------------------------------------------------- |
+| <Arg required={true} type="vec<Binary>">data</Arg> | <Input id="data" format={InputFormats.Base64}/> | The price update data for the contract to verify.          |
+| <Arg required={true} type="Coin">fee</Arg>         | <Input id="fee" format={InputFormats.BigInt}/>  | The update fee expressed in the minimal coin denomination. |
+
+</div>
+
+<Examples>
+  <Example
+    keyValues={{
+      data: async (ctx) =>
+        (await ctx.getLatestPriceFeed("Crypto.BTC/USD", "default")).vaa,
+      fee: async (ctx) => {
+        // FIXME: this gets the ETH update fee, not the cosmos fee. They're currently the same though.
+        let vaa = (await ctx.getLatestPriceFeed("Crypto.BTC/USD", "evm")).vaa;
+        return await ctx.getEthUpdateFee([vaa]);
```

Comment:
this is not great but it's hard to fix right now without adding a bunch of code to Example. I want to kick the can on it for now.